properties

```diff
@@ -1,6 +1,4 @@
 # PE can replace these values with on-prem host
-export GITHUB__DEFAULT__HOST=github.com
```

Comment:
this should be referenced in https://github.com/redhat-ai-dev/ai-lab-template/blob/56d58e8e9a85c47f0094c92c3112bf9243c7e3ea/skeleton/template.yaml#L172, as `${GITHUB__DEFAULT__HOST}`, same for the gitlab case

on-prem host should be supported. see RHTAP customizing template doc as example: 
https://gitlab.cee.redhat.com/software-supply-chain-documentation/rhtap/-/blob/main/topics/proc_customizing-sample-software-templates.adoc?ref_type=heads